src/ui/public/accessibility/kbn_ui_ace_keyboard_mode.less

```diff
@@ -0,0 +1,24 @@
+@import (reference) "~ui/styles/variables";
+
+.ui-ace-keyboard-hint {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  right: 0;
+  left: 0;
+  background: rgba(255, 255, 255, 0.7);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  opacity: 0;
+}
+
+.ui-ace-keyboard-hint-inactive {
+  display: none;
+}
+
+.ui-ace-keyboard-hint:focus {
+  opacity: 1;
+  border: 2px solid @globalColorBlue;
+  z-index: 1000;
+}
```

Comment:
Can we reformat this to use camel case per the style guide and to nest these states to be more efficient? I also added `flex-direction` and `text-align` to refine the appearance.

```less

.uiAceKeyboardHint {
  position: absolute;
  top: 0;
  bottom: 0;
  right: 0;
  left: 0;
  background: rgba(255, 255, 255, 0.7);
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  text-align: center;
  opacity: 0;

  &.uiAceKeyboardHint-isInactive {
    display: none;
  }

  &:focus {
    opacity: 1;
    border: 2px solid @globalColorBlue;
    z-index: 1000;
  }
}
```